native/cocos/scene/DrawBatch2D.h

```diff
@@ -25,19 +25,52 @@
 
 #pragma once
 
+#include "base/RefCounted.h"
+//#include "core/scene-graph/Layers.h"
+#include "core/scene-graph/Node.h"
 #include "renderer/gfx-base/GFXDef-common.h"
+#include "scene/Define.h"
 
 namespace cc {
+
+class Material;
+
 namespace scene {
 
 class Pass;
+class Model;
+
+class DrawBatch2D : public RefCounted {
```

Comment:
Please add final if it can not be inherited.

Reply:
done